BlazarBase/src/main/java/com/hubspot/blazar/base/metrics/StateToActiveBranchBuildCountPair.java

```diff
@@ -0,0 +1,45 @@
+package com.hubspot.blazar.base.metrics;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.hubspot.blazar.base.RepositoryBuild;
+
+public class StateToActiveBranchBuildCountPair {
```

Comment:
Just a proposal: 
What about naming the pair classes ActiveBranchBuildsInState, ActiveInterProjectBuildsInState, ActiveModuleBuildsInState respectively? Alternatively we could drop the Pair suffix from the class names.